src/System.Memory/tests/ReadOnlySpan/AsReadOnlySpan.cs

```diff
@@ -102,7 +102,7 @@ public static void ZeroLengthArraySegmentAsReadOnlySpan()
         public static void StringAsReadOnlySpanNullary()
         {
             string s = "Hello";
-            ReadOnlySpan<char> span = s.AsReadOnlySpan();
+            ReadOnlySpan<char> span = s.AsSpan();
```

Comment:
Should we rename the test file now that we don't have AsReadOnlySpan anymore?

Reply:
It's only the string methods that were renamed. This file is testing all the AsROSpan variants.